openqa-mail-notification/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "2.1"
```

Comment:
we are locked on docker-compose file 2.0. Please change it and see if it still works

Reply:
On it

Reply:
Done